### PR TITLE
py39: add PEP 616 methods

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

File: stdlib/2and3/builtins.pyi [hunk 1]
```diff
@@ -473,6 +473,9 @@ class str(Sequence[str], _str_base):
         @overload
         def partition(self, __sep: unicode) -> Tuple[unicode, unicode, unicode]: ...
         def replace(self, __old: AnyStr, __new: AnyStr, __count: int = ...) -> AnyStr: ...
+    if sys.version_info >= (3, 9):
+        def removeprefix(self, __prefix: bytes) -> bytes: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/builtins.pyi:477]: I think you mean str here

File: stdlib/2and3/builtins.pyi [hunk 2]
```diff
@@ -708,6 +714,9 @@ class bytearray(MutableSequence[int], ByteString):
     def lower(self) -> bytearray: ...
     def lstrip(self, __bytes: Optional[bytes] = ...) -> bytearray: ...
     def partition(self, __sep: bytes) -> Tuple[bytearray, bytearray, bytearray]: ...
+    if sys.version_info >= (3, 9):
+        def removeprefix(self, __prefix: bytes) -> bytes: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/builtins.pyi:718]: Returns a bytearray

**Comment by hauntsaninja** [comment]
oops

**Comment by hauntsaninja** [comment]
Technically, for bytes and bytearray, they accept any bytes-like argument. But I believe there isn't a good way to type that.

File: stdlib/2and3/builtins.pyi [hunk 3]
```diff
@@ -708,6 +714,9 @@ class bytearray(MutableSequence[int], ByteString):
     def lower(self) -> bytearray: ...
     def lstrip(self, __bytes: Optional[bytes] = ...) -> bytearray: ...
     def partition(self, __sep: bytes) -> Tuple[bytearray, bytearray, bytearray]: ...
+    if sys.version_info >= (3, 9):
+        def removeprefix(self, __prefix: bytearray) -> bytearray: ...
```
Comment (JelleZijlstra) [inline stdlib/2and3/builtins.pyi:718]: The argument type should actually be bytes. Typecheckers assume that bytearray is a subclass of bytes.

```
In [12]: ba = bytearray(b'xyz')                                                                                                                                                      

In [13]: ba.removeprefix(b'x')                                                                                                                                                       
Out[13]: bytearray(b'yz')

In [14]: b'xyz'.removeprefix(bytearray(b'x'))                                                                                                                                        
Out[14]: b'yz'
```

**Pull request merged**